src/server.c

```diff
@@ -316,6 +316,10 @@ struct redisCommand redisCommandTable[] = {
      "write fast @list",
      0,NULL,1,1,1,0,0,0},
 
+    {"lmpop",lmpopCommand,-3,
+     "write @list @blocking",
+     0,NULL,0,0,1,0,0,0},
```

Comment:
looks like we need a getkeys proc. one based on `genericGetKeys`.
```suggestion
     0,lmpopGetKeys,0,0,0,0,0,0},
```
@guybe7 FYI, i suppose this is no news..